src/ng/http.js

```diff
@@ -11,6 +11,8 @@ var JSON_PROTECTION_PREFIX = /^\)\]\}',?\n/;
 
 function paramSerializerFactory(jQueryMode) {
 
+  return jQueryMode ? jQueryLikeParamSerializer : paramSerializer;
```

Comment:
They only share `serializeValue`, maybe move a little more logic to this function but keep it as is?